packages/sanity/src/core/config/prepareConfig.tsx

```diff
@@ -127,7 +127,26 @@ export function prepareConfig(
     const sources = [rootSource as SourceOptions, ...nestedSources].map(({plugins, ...source}) => {
       return {
         ...source,
-        plugins: [...(plugins ?? []), ...getDefaultPlugins(defaultPluginsOptions, plugins)],
+        plugins: [...(plugins ?? []), ...getDefaultPlugins(defaultPluginsOptions, plugins)]
+          /*
+           * @FIXME: with the introduction of global references, @sanity/assist broke
+           * As a quickfix the plugins was released with a know property on the plugin definition.
+           * This checks for that property: if it is missing, the plugin is not compatible with this version of the studio.
+           * This ensurs auto updating studios can start, albeit without assist, it it is old.
```

Comment:
small typo, but apart from that this looks good

```suggestion
           * This ensures auto updating studios can start, albeit without assist, it it is old.
```